Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -170,10 +172,12 @@ struct DisplayHymnBottomBar: View {
                                 })
                             }) + [.cancel()])
                 }
-            }.sheet(item: $sheet) { tab -> ShareSheet in
+            }.sheet(item: $sheet) { tab -> AnyView in
                 switch tab {
                 case .share:
-                    return ShareSheet(activityItems: [self.viewModel.shareableLyrics])
+                    return ShareSheet(activityItems: [self.viewModel.shareableLyrics]).eraseToAnyView()
+                case .tags:
+                    return TagSheetView(title: self.viewModel.title, hymnIdentifier: self.viewModel.identifier).eraseToAnyView()
```

Comment:
I'd prefer if you just pass in the identifier and use the Repository to get the title. We don't want to create the idea that we can start passing params that we can fetch from the repo

Reply:
Addressed in 783aae0